### DISCUSSION TITLE
Save & print modified PDFs (no equations or math)

### DISCUSSION THREAD

**Discussion by ALedl** [knowledgefutures/pubpub#1368] [Support]
Hello everybody, 

our readers tell us about problems to modify (e.g. annotate, change text in) PDFs created with PubPub and then save them (error code 135, Adobe). Or they may not be able to print them. Have others encountered such problems as well? I would be very happy about any hints.

e.g. https://0277.pubpub.org/pub/uvyp13ae/release/1 

Best regards and thanks in advance,

Andreas

**Comment by gabestein** [comment]
Hi Andreas — yes, see this discussion thread: https://github.com/pubpub/pubpub/discussions/1288

Long story short, it appears to be related to a recently introduced bug in Adobe that we don't have much control over. And we're not entirely sure why it happens for some Pubs and not others, though we're still working on it.

There are a few workarounds that we've found in the Adobe help forums. One is to open them in another program like Apple's Preview and then re-export them. Another is to add a page to the PDF in Acrobat and then save it.
